brave/brave-core
# Adds p2p card

Resolves https://github.com/brave/brave-browser/issues/8620

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- enable rewards
- connect uphold account
- go to rewards page
- make sure that you see p2p card

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
